QuizApp.xcodeproj/project.xcworkspace/xcuserdata/ivankulundzic.xcuserdatad/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
/xcuserdata should be a part of .gitignore. Please add it there and check how to exclude these files ;)